upgrade_tests/test_scenarios/test_capsule.py

```diff
@@ -43,9 +43,9 @@ class Scenario_capsule_sync(TestCase):
     """
     cls_name = 'Scenario_capsule_sync'
     sat_host = get_satellite_host()
+    env.host_string = sat_host
```

Comment:
Same as above, it is needed for most modules

Reply:
Same as above answer